src/components/css/booklist.css

```diff
@@ -0,0 +1,176 @@
+* {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.container-all {
+  width: 90%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+}
+
+.container-item {
+  display: flex;
+  flex-direction: row;
+  background-color: white;
+  border: 1px solid #e7e8e8;
+  gap: 14px;
+  padding: 2rem 1.188rem 1.625rem 1.688rem;
+}
+
+.container-item:nth-child(1) {
+  margin-top: 2rem;
+}
+
+.first-item {
+  flex: 2;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
```

Comment:
- [ ] You have done a pretty good job here 👍👍 However, as mentioned by the previous reviewer, kindly adjust the spacing according to how it is given in the design guidelines. Can you please increase it a bit more?  😃See the difference in the images below 👇


Your design | Given design 
---| ---| 
![image](https://github.com/AliShahSafdari/bookstore/assets/106119105/fa135d8d-2bd2-4575-8b2f-1aad4965fbe2) |![image](https://github.com/AliShahSafdari/bookstore/assets/106119105/24143341-421d-4496-adda-21bc4542d77d)

Reply:
Thanks, I add the space.